packages/veritone-widgets/test/redux/modules/veritoneApp/saga.test.js

```diff
@@ -0,0 +1,32 @@
+import { put, call, select } from 'redux-saga/effects';
+import { cloneableGenerator } from 'redux-saga/utils';
+import { removeFunctions } from '../../../helpers/utils';
+import {
+  handleAppAuth,
+  watchAppAuth,
+} from '../../../../src/redux/modules/veritoneApp/saga';
+import * as appModule from '../../../../src/redux/modules/veritoneApp';
+import _ from 'lodash';
+import { modules } from '../../../../../veritone-redux-common/dist/bundle-es.js';
+const { user: userModule, auth: authModule } = modules;
+const rF = removeFunctions;
```

Comment:
fyi this isn't needed - you can define an alias when you import i.e `import { removeFunctions as rF } from '../../../helpers/utils'`